### PR TITLE
[SDPA-7621] Minimum statistic grid validations.

### PR DESCRIPTION
**Pull request by edyuenyw**
**JIRA issue:** https://digital-vic.atlassian.net/browse/SDPAP-7621

### Changed
1.  Fixed validation on minimum of 2 statistics grid block validation.

### CODE REVIEW

File: src/Validate/StatisticsGrid.php [hunk 1]
```diff
@@ -21,9 +21,17 @@ class StatisticsGrid {
...
+      if (is_numeric($key)) {
+        $count += 1;
+      }
+    }
+
     $error = FALSE;
-    $max_delta = $element['subform']['field_statistic_block']['widget']['#max_delta'];
-    if ($max_delta == 0) {
+    if ($count < 2) {
       $error = TRUE;
```
Comment (krakerag) [inline src/Validate/StatisticsGrid.php:35]: Soooo do we have a unit test for this? This seems like a really easy thing to pass some input into and validate that your count works :)

Comment (vincent-gao) [reply src/Validate/StatisticsGrid.php]: Ed's has left , I will refactor the code and add the unit test.

**Comment by vincent-gao** [comment]
hey @krakerag 
I have made some changes to the code by refactoring it and also included a unit test.

**Review by krakerag — APPROVED** [review]
Good test - thank you!

**Pull request merged**